aunt-leahs-app/src/pages/volunteerPage/volunteerPage.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../../App.css';
+import SelectBox from '../../components/select-box/select-box.jsx'
+import CustomButton from '../../components/customButton/customButton.jsx'
+import Head from '../../components/header/header.jsx'
+
+const VolunteerPage = () => (
+	<div className="App">
+		<Head page="Volunteer Login" />
+		<div style={{ margin: 'auto', width: '200px' }}>
+			<SelectBox
+				name="Existing Volunteer"
+				items={[
+					{ value: 'Viniel Kumar', id: 1 },
+					{ value: 'Pritpal Chauhan', id: 2 },
+					{ value: 'John Doe', id: 3 },
+					{ value: 'Justin Kwan', id: 4 },
+					{ value: 'Cody Thechange', id: 5 }
+				]}
+			/>
+		</div>
+		<br />
```

Comment:
Use css styles for line spacing instead of <br />

Reply:
Not too sure how to import css styling